Mbody_example.py

```diff
@@ -114,59 +101,59 @@
 spike_times = spike_times.repeat([len(p) for p in sorted_variants])
 spike_indices = np.concatenate(sorted_variants)
 
-PN = SpikeGeneratorGroup(N_AL, spike_indices, spike_times)
+PN = SpikeGeneratorGroup(N_PN, spike_indices, spike_times)
 
 # iKC of the mushroom body
-I_syn = '''I_syn = g_PN_iKC*(V - E_e): amp
-           dg_PN_iKC/dt = -g_PN_iKC/tau_PN_iKC : siemens'''
+I_syn = '''g_PNiKC : siemens
```

Comment:
I think removing the decay of the synaptic conductance was probably not intended here?

Reply:
Indeed not - I have no clue how this happened as I was only doing query-replace in emacs for the variable names ... a mystery. Anyway - I think I have reverted to what it should be (please check).